### PR TITLE
(docs): Replace `@RequestMapping` with newer `@GetMapping` (etc.)

### PR DESCRIPTION
**Pull request by AlexElin**
See the subject

### CODE REVIEW

File: docs/modules/ROOT/pages/spring-cloud-openfeign.adoc [hunk 1]
```diff
@@ -39,16 +39,16 @@ public class Application {
 ----
 @FeignClient("stores")
 public interface StoreClient {
-	@RequestMapping(method = RequestMethod.GET, value = "/stores")
```
Comment (OlgaMaciaszek) [inline docs/modules/ROOT/pages/spring-cloud-openfeign.adoc:42]: Please leave this one as `@RequestMapping` - like this we showcase various possible options.

**Review by OlgaMaciaszek — CHANGES_REQUESTED** [review]
Hello @AlexElin, thanks for submitting the PR. Have added a comment. Please address it.

File: docs/modules/ROOT/pages/spring-cloud-openfeign.adoc (hunk 1 above)
Comment (AlexElin) [reply docs/modules/ROOT/pages/spring-cloud-openfeign.adoc:42]: changed

**Review by OlgaMaciaszek — APPROVED** [review]
Thanks @AlexElin. LGTM.

**Pull request merged**